cmd/devstream/apply.go

```diff
@@ -19,6 +19,7 @@ DevStream will generate and execute a new plan based on the config file and the
 }
 
 func applyCMDFunc(cmd *cobra.Command, args []string) {
+	initLogConf()
```

Comment:
There should be a more appropriate place to add this code instead of calling it at the entry point of all subcommands